pages/common/pueue-completions.md

```diff
@@ -0,0 +1,16 @@
+# pueue completions
+
+> Generates shell completion files for bash, elvish, fish, power-shell, zsh.
+> More information: <https://github.com/Nukesor/pueue>.
+
+- Generate completions for bash on Linux:
+
+`sudo pueue completions bash {{/usr/share/zsh/site-functions}}`
```

Comment:
Oops&mdash;looks like you used the `zsh` path for the `bash` completions. Will you please fix this?